appengine/bigquery/requirements.txt

```diff
@@ -1 +1 @@
-google-api-python-client
+google-api-python-client==1.4.2
```

Comment:
alright fine with me...its an age-old debate in other contexts too (static vs dynamic linkins is similar), advantages and disadvantages each way but if you want to pin, fine by me